lib/exosuit/instance.rb

```diff
@@ -44,5 +44,9 @@ def self.all
     def self.running
       all.select { |i| i.state.name == 'running' }
     end
+
+    def self.latest_ami
+      @ami = Exosuit.ssm.get_parameters({ names: [IMAGE_NAME] }).parameters[0].value
```

Comment:
If this is the one and only place we use SSM, how about we instead do this?

```ruby
Aws::SSM::Client.new(profile: profile_name)
  .get_parameters(names: [IMAGE_NAME])
  .parameters[0]
  .value
```

Then we don't have to have the `Exosuit#ssm` method which may never be used anywhere else.